docs/operations-guide/running-metabase-on-docker.md

```diff
@@ -4,164 +4,206 @@ Metabase provides an official Docker image via Dockerhub that can be used for de
 
-The migration process involves 2 main steps:
+Here's how to use a database file, owned by your account and stored in your home directory:
 
-1. Stop the existing Metabase container
-2. Run a new, temporary Metabase container to perform the migration
+```
+docker run -d -v ~/my-metabase-db:/metabase.db --name metabase -e MB_DB_FILE=/metabase.db -e MUID=$UID -e MGID=$GID -p 3000:3000 metabase/metabase
+```
 
-Using a Postgres container as the target, here's an example invocation:
+### Setting the Java Timezone
 
-    docker run --name metabase-migration \
-        -v /path/metabase/data:/metabase-data \
-        -e "MB_DB_FILE=/metabase-data/metabase.db" \
-        -e "MB_DB_TYPE=postgres" \
-        -e "MB_DB_DBNAME=metabase" \
-        -e "MB_DB_PORT=5432" \
-        -e "MB_DB_USER=<username>" \
-        -e "MB_DB_PASS=<password>" \
-        -e "MB_DB_HOST=my-database-host" \
-        metabase/metabase load-from-h2
+It's best to set your Java timezone to match the timezone you'd like all your reports to come in. You can do this by simply specifying the `JAVA_TIMEZONE` environment variable which is picked up by the Metabase launch script. For example:
 
-To further explain the example: in addition to specifying the target database connection details, set the `MB_DB_FILE` environment variable for the source H2 database location, and pass the argument `load-from-h2` to begin migrating.
+```
+docker run -d -p 3000:3000 \
+  -e "JAVA_TIMEZONE=US/Pacific" \
+  --name metabase metabase/metabase
+```
 
-### Setting the Java Timezone
+### Copying the application database
 
-It's best to set your Java timezone to match the timezone you'd like all your reports to come in.  You can do this by simply specifying the `JAVA_TIMEZONE` environment variable which is picked up by the Metabase launch script.  For example:
+The default location for the application database in the container is `/metabase.db/metabase.db.mv.db`. You can copy this directory out of the container using the following command (replacing `CONTAINER_ID` with the actual container ID or name, `metabase` if you named the container):
 
-    docker run -d -p 3000:3000 \
-      -e "JAVA_TIMEZONE=US/Pacific" \
-      --name metabase metabase/metabase
+```
+docker cp CONTAINER_ID:/metabase.db ./
+```
 
+The DB contents will be left in a directory named metabase.db.
 
-### Additional custom settings
+Some older versions of Metabase stored their db in a different default location:
 
-While running Metabase on docker you can use any of the custom settings from [Customizing the Metabase Jetty Webserver](customizing-jetty-webserver.md) by setting environment variables on your docker run command.
+```
+docker cp CONTAINER_ID:/metabase.db.mv.db metabase.db.mv.db
+```
 
```

Comment:
I don't know which versions has the H2 file in a different location, but sounds like it's a very long time ago, so let's remove.